lms/templates/help_modal.html

```diff
@@ -201,29 +212,32 @@
     $("#feedback_link_problem").click(function(event) {
         showFeedback(
             event,
-            "${_('problem')}",
-            "${_('Report a Problem')}",
-            "${_('Brief description of the problem')}" + "*",
-            "${_('Details of the problem you are encountering')}" + "*" + "<span class='tip'>" +
-              "${_('Include error messages, steps which lead to the issue, etc.')}" + "</span>"
+            "${_('problem') | n, js_escaped_string}",
+            "${_('Report a Problem') | n, js_escaped_string}",
+            "${_('Brief description of the problem') + '*' | n, js_escaped_string}" ,
+            "${Text(_('Details of the problem you are encountering{asterisk}{begin_span}Include error messages, steps which lead to the issue, etc.{end_span}')).format(
+                asterisk='*',
+                begin_span=HTML('<span class=tip>'),
+                end_span=HTML('</span>'),
+            ) | n, js_escaped_string}"
         );
     });
     $("#feedback_link_suggestion").click(function(event) {
         showFeedback(
             event,
-            "${_('suggestion')}",
-            "${_('Make a Suggestion')}",
-            "${_('Brief description of your suggestion')}" + "*",
-            "${_('Details')}" + "*"
+            "${_('suggestion') | n, js_escaped_string}",
+            "${_('Make a Suggestion') | n, js_escaped_string}",
+            "${_('Brief description of your suggestion') + '*' | n, js_escaped_string}",
+            "${_('Details') + '*' | n, js_escaped_string}"
         );
     });
     $("#feedback_link_question").click(function(event) {
         showFeedback(
             event,
-            "${_('question')}",
-            "${_('Ask a Question')}",
-            "${_('Brief summary of your question')}" + "*",
-            "${_('Details')}" + "*"
+            "${_('question') | n, js_escaped_string}",
+            "${_('Ask a Question') | n, js_escaped_string}",
+            "${_('Brief summary of your question') + '*' | n, js_escaped_string}",
+            "${_('Details') + '*' | n, js_escaped_string}"
```

Comment:
Does operator precedence work here the way you want it to? (Not a mako expert by any stretch)